packages/netlify-cms/webpack.config.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 require('dotenv').config({ path: path.join(__dirname, '..', '..', '.env') });
 const webpack = require('webpack');
-const FriendlyErrorsWebpackPlugin = require('friendly-errors-webpack-plugin');
+const FriendlyErrorsWebpackPlugin = require('@soda/friendly-errors-webpack-plugin');
```

Comment:
- `friendly-errors-webpack-plugin` does not support `webpack@5` and is [archived on GitHub](https://github.com/geowarin/friendly-errors-webpack-plugin).
- `@soda/friendly-errors-webpack-plugin` is a fork which supports `webpack@5`.